src/Modules/AuthModule/Components/ForgetPassword/ForgetPassword.tsx

```diff
@@ -1,3 +1,214 @@
+
 export default function ForgetPassword() {
-  return <div>ForgetPassword</div>;
+  const [isDark, setIsDark] = useState(() => {
+    const value = localStorage.getItem("theme");
+    if (value === "dark" || value === null) return true;
+    return false;
+  });
+  const [spinner, setSpinner] = useState<boolean>(false);
+  const [isClicked, setIsClicked] = useState<boolean>(false);
+  const navigate = useNavigate();
+
+  const signUpWaitToast = {
+    onClose: () => setIsClicked(false),
+  };
+
+  const {
+    handleSubmit,
+    control,
+    formState: { errors },
+  } = useForm<FormData>();
+
+  const onSubmit = async (data: FormData) => {
+    setSpinner(true);
+    setIsClicked(true);
+    try {
+      const res = await axios.post(
+        `${getBaseUrl()}/api/v0/admin/users/forgot-password`,
+        data
+      );
+      setSpinner(false);
+      toast.success(res.data.message, signUpWaitToast);
+      navigate("/auth/reset-password");
+    } catch (error) {
+      if (axios.isAxiosError(error) && error.response) {
+        toast.error(
+          error.response.data.message || "fail signin",
+          signUpWaitToast
+        );
+      }
+      setSpinner(false);
+    }
+  };
+
+  return (
+    <>
+      <Box
+        component={"section"}
+        minHeight={"100vh"}
+        sx={{ pt: { xs: "0px", md: "24px" } }}
+      >
+        <Box
+          sx={{ display: { xs: "none", md: "block" } }}
+          left={50}
+          top={30}
+          position={"absolute"}
+        >
+          <img width={"150px"} src={isDark ? logoDark : logoLight} alt="logo" />
+        </Box>
+        <Box
+          sx={{ display: { xs: "block", md: "none" } }}
+          left={50}
+          top={30}
+          position={"absolute"}
+        >
+          <img width={"150px"} src={logoDark} alt="logo" />
+        </Box>
+        <Grid
+          container
+          height={"96vh"}
+          display={"flex"}
+          justifyContent={"center"}
+          alignItems={"center"}
+        >
+          <Grid
+            width={"100%"}
+            height={"100%"}
+            item
+            display={"flex"}
+            justifyContent={"center"}
+            alignItems={"center"}
+            flexDirection={"column"}
+            xs={12}
+            md={6}
+            className={`${Style.forgetPass}`}
+          >
+            <Box width={{ md: "65%", xs: "90%" }}>
+              <Box>
+                <Typography mb={3} fontWeight={"500"} variant="h5">
+                  Forgot password
+                </Typography>
+                <Typography fontWeight={"500"} variant="body2">
+                  If you already have an account register
+                </Typography>
+                <Typography
+                  display={"flex"}
+                  alignItems={"center"}
+                  fontWeight={"500"}
+                  variant="body2"
+                >
+                  You can
+                  <Button variant="text" color={"error"}>
```

Comment:
We shouldn't have nested interactive elements. The button is used for styling purposes only. We can give it the component prop and then pas the Link component to it. This will make the underlying element for the button to be the anchor element as the Link.